lisa/schema.py

```diff
@@ -546,12 +573,14 @@ def _generate_min_capability(self, capability: Any) -> Any:
             )
         else:
             raise LisaException("memory_mb cannot be zero")
-        if self.disk_count or capability.disk_count:
-            min_value.disk_count = search_space.generate_min_capability_countspace(
-                self.disk_count, capability.disk_count
+        if self.data_disk_count or capability.data_disk_count:
+            min_value.data_disk_count = search_space.generate_min_capability_countspace(
+                self.data_disk_count, capability.data_disk_count
+            )
+        if self.data_disk_iops or capability.data_disk_iops:
```

Comment:
The disk iops cannot be done here, because it's not an integer on Azure. In Azure, we cannot select any integer as iops, it depends on the data_size. So the min capability must be resolved in Azure platform.